.mailmap

```diff
@@ -831,7 +831,7 @@ Ondřej Čertík <ondrej@certik.cz> <ondrej.certik@gmail.com>
 Ondřej Čertík <ondrej@certik.cz> <ondrej@certik.us>
 Ondřej Čertík <ondrej@certik.cz> ondrej.certik <devnull@localhost>
 Or Dvory <gidesa@gmail.com>
-OrestisVaggelis <orestis22000@gmail.com>
+OrestisVaggelis <orestis22000@gmail.com> Orestis Vaggelis <orestisvaggelis@mail.com>
```

Comment:
Which name and email do you want to appear in the AUTHORS file?

Reply:
I'd prefer Orestis Vaggelis <orestisvaggelis@mail.com> to appear, but without the first one, I was getting an error during the tests.

Reply:
Then the above should be reversed like:
```
Orestis Vaggelis <orestisvaggelis@mail.com>  OrestisVaggelis <orestis22000@gmail.com>
```
Also you should remove the line `OrestisVaggelis <orestis22000@gmail.com>` from the AUTHORS file. The new name will then be added to AUTHORS automatically at the time of the next release.